debian/patches/Do-not-build-the-tests

```diff
@@ -0,0 +1,34 @@
+Description: <short summary of the patch>
+ TODO: Put a short summary on the line above and replace this paragraph
+ with a longer explanation of this change. Complete the meta-information
+ with other relevant fields (see below for details). To make it easier, the
+ information below has been extracted from the changelog. Adjust it or drop
+ it.
+ .
+ nemo-qml-plugin-dbus (2.1.24-1) UNRELEASED; urgency=medium
+ .
+   * initial release
+Author: Sebastian Spaeth <Sebastian@SSpaeth.de>
+
+---
+The information above should follow the Patch Tagging Guidelines, please
+checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
+are templates for supplementary fields that you might want to add:
+
+Origin: <vendor|upstream|other>, <url of original patch>
+Bug: <url in upstream bugtracker>
+Bug-Debian: https://bugs.debian.org/<bugnumber>
+Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
+Forwarded: <no|not-needed|url proving that it has been forwarded>
+Reviewed-By: <name and email of someone who approved the patch>
+Last-Update: 2022-08-09
+
+--- nemo-qml-plugin-dbus-2.1.24.orig/dbus.pro
++++ nemo-qml-plugin-dbus-2.1.24/dbus.pro
+@@ -1,5 +1,5 @@
+ TEMPLATE = subdirs
+-SUBDIRS += src tests doc
++SUBDIRS += src doc
```

Comment:
You can skip building the tests without patches.
Just do it like this:
```bash
for dirs in src doc ; do 
   make -C $dir
done
```

Reply:
Or perhaps better by having a .pro config option to disable building the tests and then using that in the packaging. The project having patches for its own content is sort of strange.

Reply:
is a good idea i update the build rules in debian/rules